integrations/server_test.go

```diff
@@ -3173,8 +3172,6 @@ func TestServer(t *testing.T) {
 					tt.wants.body,
 				)
 			}
-
-			tt.args.server.Listener.Close()
```

Comment:
why do we not need to close the server listener?

Reply:
It get's closed in the function that created it - https://github.com/influxdata/chronograf/pull/5342/files#diff-91bbeda7eb98a7adc57b9e47e2cf5c2bR414